packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/representation/DiagramRepresentation.tsx

```diff
@@ -96,7 +146,7 @@ export const DiagramRepresentation = ({
   if (state.complete) {
     return <div>The representation is not available anymore</div>;
   }
-  if (!state.diagramRefreshedEventPayload) {
+  if (!state.diagramRefreshedEventPayload || !state.diagramDescription) {
```

Comment:
This adds a whole new query, with the corresponding latency, to the diagram rendering.